packages/eos-components/src/components/GridPage.vue

```diff
@@ -28,4 +28,11 @@ export default {
 
   @import '../styles.scss';
 
+  .fade-enter-active, .fade-leave-active {
+    transition: all 0.3s;
```

Comment:
If we start adding transitions I think it's better to use the theme. Bootstrap has:
```
  .fade-enter-active, .fade-leave-active {
    @include transition($transition-fade);
  }
```
Then Joana can decide if it looks good or not, and we just tweak the $transition-fade variable in the theme. What do you think?

Reply:
Yes, It's a good idea to move the transition to the theme. I have not played a lot with transitions, it could be better to implement an slide down animation or something like that, but the fade-in-out was the easier thing to implement, and I thought it could be sufficient.